import-dashboards.py

```diff
@@ -224,39 +225,47 @@ def add_datasources(api_key):
 
 
 def add_panels():
-    for app in ['panels']:
-        print ' * Adding %r' % (app,)
-        source_dir = '/usr/share/percona-dashboards/' + app
-        dest_dir = '/var/lib/grafana/plugins/'
-        if os.path.isdir(source_dir):
-            files_list = os.listdir(source_dir)
-            print '  * Copying %r' % (app,)
-            try:
-                shutil.rmtree(dest_dir, False)
-            except Exception as err:
-                print '  * Failed to remove %s: %s' % (dest_dir, err)
-            shutil.copytree(source_dir, dest_dir)
-            print '  * Unzipping %r' % (app,)
+    print ' * Adding panels'
+    if os.path.isdir(GRAFANA_SOURCE_PLUGINS_DR):
+        files_list = os.listdir(GRAFANA_SOURCE_PLUGINS_DR)
+        print '  * Copying panels'
+        if not os.path.isdir(GRAFANA_PLUGINS_DR):
+            shutil.copytree(GRAFANA_SOURCE_PLUGINS_DR, GRAFANA_PLUGINS_DR)
+        else:
```

Comment:
According to copytree doc https://docs.python.org/3/library/shutil.html#shutil.copytree , there is a dirs_exist_ok parameter we can use and keep only the shutil.copytree part and remove the else and the for loop.
Is that correct or there is another reason to have the for loop?